src/components/Layout/Layout.scss

```diff
@@ -1,7 +1,8 @@
 @use "../../styles/variables" as *;
 
 .layout {
+
```

Comment:
This is very small but please try remove whitespace if it wasn't there before. Very small but if everyone leaves different white spaces it will cause merge errors later.